src/types/Error.ts

```diff
@@ -0,0 +1,8 @@
+export enum Error {
+  None = '',
+  Add = 'Unable to add a todo',
+  Delete = 'Unable to delete a todo',
+  Update = 'Unable to update a todo',
+  Title = 'Tittle can not be empty',
+  Load = 'Can not load todos',
+}
```

Comment:
It's better to avoid such usage of enums. Better way is:
```suggestion
export enum Error {
  None = 'NO_ERROR',
  Add = 'ADD_ERROR',
  Delete = 'DELETE_ERROR',
  Update = 'UPDATE_ERROR',
  Title = 'EMPTY_TITLE_ERROR',
  Load = 'LOADING_ERROR',
}

export type ErrorMessages = Record<Error, string>;

// this needs to be in `constants.ts`
export const errorMessages: ErrorMessages = {
  [Error.None]: '',
  [Error.Add]: 'Unable to add a todo',
  [Error.Delete]: 'Unable to delete a todo',
  [Error.Update]: 'Unable to update a todo',
  [Error.Title]: 'Tittle can not be empty',
  [Error.Load]: 'Can not load todos',
}
```